lib/powerpoint.js

```diff
@@ -228,13 +423,14 @@ class Powerpoint extends EventEmitter2 {
 
   /**
    * @param sortedImages sorted by file and page/image number
-   * @returns {array} of files from sortedImages that are the last image for each file
+   * @returns {array} of files from sortedImages that are the last image for
```

Comment:
Same thing: I thought it was Array with a capital A

Reply:
I can do it that way, I think JSDoc figures it out either way